### PR TITLE
Add test stubs for archive and publish

### PR DESCRIPTION
**Pull request by willingc**
There's still work to do with the async tests. I don't plan to add to this PR so please review as is.

### CODE REVIEW

File: bookstore/tests/test_publish.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import pytest
+
+
+def test_hello():
+    assert 1000
```
Comment (MSeal) [inline bookstore/tests/test_publish.py]: This seem erroneous?

Comment (willingc) [reply bookstore/tests/test_publish.py]: Ah...this was to keep pytest from complaining about the empty file. So I threw in this test which should get removed when we have real publish tests ;-)

Comment (willingc) [reply bookstore/tests/test_publish.py]: Let me remove and replace with something.

**Comment by MSeal** [comment]
Thanks for adjusting the tests

**Pull request merged**